test/e2e/main.http.test.ts

```diff
@@ -42,6 +42,7 @@ describe('http/main', () => {
     await server.pubsub.subscribe('pubsub-redis', 'test-topic', mockPubSubSubscribe);
 
     // Start server
+    await client.start();
```

Comment:
Should we update the docs too?

Reply:
And examples.

Reply:
Added those, will do a bigger pass later on as well